### PR TITLE
Feature assert tolerance

### PR DESCRIPTION
**Pull request by janousch**
Exposed the tolerance parameter on the Equal/Not Equal functions.
Removed unused includes.

### CODE REVIEW

File: DaedalicTestAutomationPlugin/Source/DaedalicTestAutomationPlugin/Private/DaeTestAssertBlueprintFunctionLibrary.cpp [hunk 1]
```diff
@@ -4,7 +4,6 @@
 #include "DaeTestTriggerBox.h"
 #include <Blueprint/UserWidget.h>
 #include <Components/Image.h>
-#include <Components/PanelWidget.h>
```
Comment (janousch) [inline DaedalicTestAutomationPlugin/Source/DaedalicTestAutomationPlugin/Private/DaeTestAssertBlueprintFunctionLibrary.cpp:7]: Resharper marks this include as unused, but it's necessary to compile the plugin.

**Comment by npruehs** [comment]
Thanks for the PR! :)

File: DaedalicTestAutomationPlugin/Source/DaedalicTestAutomationPlugin/Private/DaeTestAssertBlueprintFunctionLibrary.cpp (hunk 1 above)
Comment (npruehs) [reply DaedalicTestAutomationPlugin/Source/DaedalicTestAutomationPlugin/Private/DaeTestAssertBlueprintFunctionLibrary.cpp:7]: Yes, it is required: AssertWidgetIsVisible calls !IsValid(Widget->GetParent() which is of type UPanelWidget.

Without that include, I get

`1>DaeTestAssertBlueprintFunctionLibrary.cpp(501): error C2664: 'bool IsValid(const UObject *)': cannot convert argument 1 from 'UPanelWidget *' to 'const UObject *'
`

Comment (npruehs) [reply DaedalicTestAutomationPlugin/Source/DaedalicTestAutomationPlugin/Private/DaeTestAssertBlueprintFunctionLibrary.cpp:7]: Added include again in commit 6a8cf5a16b4056485aed10e338a04762dfa1a60f.

**Pull request merged**